runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -807,6 +902,145 @@ JITServerAOTCacheMap::buildCacheFileName(const std::string &cacheDir, const std:
    return cacheFileName + "JITServerAOTCache." + cacheName;
    }
 
+void
+JITServerAOTCacheMap::queueAOTCacheForSavingToFile(const std::string &cacheName)
+   {
+   OMR::CriticalSection cs(_monitor);
+   _cachesToSaveQueue.push_back(cacheName);
+   }
+
+// saveNextQueuedAOTCachetoFile() attempts to save to file the next AOT cache that is queued for saving.
+// If a similarly named cache already exists on disk, we check whether our in-memory
+// cache has more methods, and if it does, we overwrite the existing cache with ours
+// At the end, we update the time when the last save attempt for this cache was performed
+// (so that we don't try too often) and the number of AOT methods that were saved to file.
+// This method acquires the AOTCacheMap monitor and _cacheMethodMonitor for the cache
```

Comment:
Typo: `_cachedMethodMonitor`. This is also a bit confusing because this method does not acquire it directly.

Reply:
Changed the comment to reflect that. I felt the message is required because I find it hard to read all callees to find what monitors can be acquired by a particular function.